fern/products/sdks/overview/go/publishing-to-go-package-manager.mdx

```diff
@@ -42,15 +41,15 @@ you'll have a versioned package published on Pkgsite.
     ```yaml {2-11}
     groups:
       go-sdk:
-      generators:
-        - name: fernapi/fern-go-sdk
-          version: <Markdown src="/snippets/version-number.mdx"/>
-          output:
-            location: local-file-system
-            path: ../sdks/go
-          config:
-            module:
-            path: sdk
+        generators:
+          - name: fernapi/fern-go-sdk
+            version: <Markdown src="/snippets/version-number.mdx"/>
+            output:
+              location: local-file-system
+              path: ../sdks/go
+            config:
+              module:
+              path: sdk
```

Comment:
The `path` key should be indented -- I could also see us removing the `module` config block from the docs entirely in the future, but it's best to at least keep it consistent with the behavior of the `fern` CLI.